enum.go

```diff
@@ -140,6 +158,32 @@ func possibleEnumMember(constName *ast.Ident, info *types.Info) (et enumType, na
 	return enumType{tn}, obj.Name(), determineConstVal(constName, info), true
 }
 
+func findIgnoredTypes(pass *analysis.Pass, inspect *inspector.Inspector, info *types.Info) map[types.Type]struct{} {
+	ignoredTypes := map[types.Type]struct{}{}
+
+	inspect.Preorder([]ast.Node{&ast.GenDecl{}}, func(n ast.Node) {
+		gen := n.(*ast.GenDecl)
+		if gen.Tok != token.TYPE {
+			return
+		}
+
+		doIgnoreDecl := isIgnoreDecl(pass, gen.Doc)
+
```

Comment:
Minor: For consistency with the code in func findEnums, consider changing this to an early return:

```go
if isIgnoreDecl(pass, gen.Doc) {
	return
}
```

and then `doIgnoreDecl ||` inside the loop can be removed.

Reply:
Ignore my previous comment. In hindsight I realize an early return might not be possible because the ignored types have to be collected in the loop.